### DISCUSSION TITLE
Is possible to embed slide pack (e.g. a power point presention) in Quartro Blog ?

### DISCUSSION THREAD

**Discussion by msh855** [quarto-dev/quarto-cli#6888] [Q&A]
### Description

I want to use Quarto to publish a blog post with an embedded Power Point presentation. I was looking for information in the documentation on how to do this, but I couldn't find it.

Could you please help me in determining whether or not I can do this? 

Thank you

**Comment by mcanouil** [comment]
Microsoft documented this: <https://support.microsoft.com/en-us/office/embed-a-presentation-in-a-web-page-or-blog-19668a1d-2299-4af3-91e1-ae57af723a60>.

Note that this is not really related to Quarto itself but rather to HTML.

**Reply by msh855** [reply]
Thank you, I would have a look and try. 

I am understanding the steps correctly then: 

1. Do as the Microsoft link says to get the link for embedding.
2. Use this link in a Quarto document file with an HTML code chunk where I will paste this link 

is this is the way ?

**Reply by mcanouil** [reply]
In theory and possibly in a raw html block ` ```{=html} ` to avoid Quarto/Pandoc to modify the HTML component from Microsoft documentation.
